cmd/argo/commands/archive/list.go

```diff
@@ -25,6 +25,18 @@ func NewListCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "list",
 		Short: "list workflows in the archive",
+		Example: `# List all Archived Workflows:
+  argo archive list
+  
+# List Archived Workflows in YAML format:
+  argo archive list --output yaml
+  
+# List Archived Workflows with a specific chunk size:
+  argo archive list --chunk-size <chunk_size>
```

Comment:
```suggestion
# List Archived Workflows in chunks of 20:
  argo archive list --chunk-size 20
```

This is an example, so we should be specific; otherwise it is not different from the existing option description